tls/s2n_client_hello_request.c

```diff
@@ -61,10 +68,8 @@ S2N_RESULT s2n_client_hello_request_recv(struct s2n_connection *conn)
     }
 
     s2n_renegotiate_response response = S2N_RENEGOTIATE_REJECT;
-    if (conn->config->renegotiate_request_cb) {
-        RESULT_GUARD_POSIX((conn->config->renegotiate_request_cb)(
```

Comment:
Note: We probably do this in _a lot_ of places where we call callbacks. Maybe make an issue for the rest of the callbacks?

Reply:
https://github.com/aws/s2n-tls/issues/3559